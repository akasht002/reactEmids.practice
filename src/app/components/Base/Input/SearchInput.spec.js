import React from 'react';
import { SearchInput } from './index';
import renderer from 'react-test-renderer';

 test('SearchInput Component Testing', () => {
    const component = renderer.create(
        <SearchInput
            id={'1'}
            autoComplete={false}
            placeholder={'Test SearchInput'}
            className={'Test SearchInput'}
            disabled={false}>
        </SearchInput>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});