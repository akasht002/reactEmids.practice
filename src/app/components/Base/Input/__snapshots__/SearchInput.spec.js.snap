// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput Component Testing 1`] = `
<div
  className="input-group"
>
  <span />
  <input
    autoComplete={false}
    className="Test SearchInput"
    placeholder="Test SearchInput"
  />
</div>
`;
